app/src/main/java/ch/epfl/skysync/viewmodel/CalendarViewModel.kt

```diff
@@ -63,10 +63,14 @@ class CalendarViewModel(
 
   val uiState: StateFlow<CalendarUiState> =
       combine(user, loadingCounter) { user, loadingCounter ->
+            val flights =
```

Comment:
Why retrieve the flights here ? It doesn't make sense to me, shouldn't this go in a refresh... function ?

Reply:
because it depends on the user. We want to load the assigned flights as soon as the user was successfully fetched